src/components/screens/dashboard/newsFeed/newsFeed.js

```diff
@@ -15,18 +15,15 @@ const NewsFeed = (props) => {
     newsFeed,
     t,
   } = props;
-  const filteredNewsFeed = newsFeed.data || [];
+  const filteredNewsFeed = newsFeed.data.data || [];
```

Comment:
Can we use the `withData` `transformResponse` function here to only get data instead of `data.data`?

Reply:
Some lines below it needs error `newsFeed.error` thus transformResponse would get rid of it.